recipes/ceres-solver/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set version = "1.14.0" %}
+
+package:
+  name: ceres-solver
+  version: {{ version }}
+
+source:
+  url: http://ceres-solver.org/ceres-solver-{{ version }}.tar.gz
+  sha256: 4744005fc3b902fed886ea418df70690caa8e2ff6b5a90f3dd88a3d291ef8e8e
+
+build:
+  number: 0
+  skip: true  # [win and vc<14]
```

Am I right in thinking this will always be used as a shared library? If so:
```suggestion
  skip: true  # [win and vc<14]
  run_exports: 
    - {{ pin_subpackage('ceres-solver', max_pin='x.x') }}
```
https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html?highlight=selectors#export-runtime-requirements